### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by panjuksergy**
Here u can find fixed Header with my designed footer

### CODE REVIEW

File: src/basic.css [hunk 1]
```diff
@@ -214,13 +211,14 @@ label .menu {
...
 
 label .hamburger {
-  position: absolute;
-  top: 135px;
-  left: 50px;
+  position: relative;
+
   width: 30px;
   height: 2px;
+  margin-top: 30px;
```
Comment (panjuksergy) [inline src/basic.css:218]: Idk how to make in another way. I think this **_hardcode_** perfect

**Comment by panjuksergy** [comment]
@DemianParkhomenko 
Could u pls check this and make review to marge this pl !

File: src/basic-adaptation.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    height: 70%;
+  }
+
+  .footer__picture {
+    width: 300px;
+  }
+}
+@media screen and (max-width: 425px) {
+  footer ul li a {
+    width: 40px;
```
Comment (DemianParkhomenko) [inline src/basic-adaptation.css:34]: Suggest:
```
   --size: 40px
    width: var(--size);
    height: var(--size);
    line-height: var(--size); 
```

**Review by DemianParkhomenko — CHANGES_REQUESTED** [review]
Adaptive layout for phones is broken: 
image
image

File: src/Pages/home.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          <div class="header__git__animation">
+            <a href="#"
+              ><img class="header__github" src="/src/Icons/Octocat.png"
+            /></a>
+          </div>
+        </div>
+
+        <label class="burger">
+          <input type="checkbox" />
+          <span class="menu"> <span class="hamburger"></span> </span>
```
[the remark this answers is not in the document]
Comment (panjuksergy) [reply src/Pages/home.html:49]: burger = ham + burger bro))

File: src/basic-adaptation.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    display: none;
+  }
+}
+@media screen and (min-width: 795px) {
+  .burger {
+    display: none;
+  }
+}
+/* Footer adapt. */
+@media screen and (max-width: 544px) {
```
[the remark this answers is not in the document]
Comment (panjuksergy) [reply src/basic-adaptation.css:16]: I changed the resolution while the site worked fine. When something went wrong, I created a new breakpoint

File: src/basic.css [hunk 5]
```diff
@@ -4,4 +4,306 @@ body {
...
+footer ul li a:hover:before {
+  top: 0;
+}
+
+footer ul li:nth-child(1) a:before {
+  background: #3b5999;
+}
+
+footer ul li:nth-child(2) a:before {
+  background: #55acee;
```
[the remark this answers is not in the document]
Comment (panjuksergy) [reply src/basic.css:177]: Ok

**Pull request merged**